inputgraph/loader.go

```diff
@@ -454,6 +458,64 @@ func (l *loader) handleArg(ctx context.Context, cmd spec.Command, isBase bool) e
 	return nil
 }
 
+func (l *loader) handleLet(ctx context.Context, cmd spec.Command) error {
+	var opts commandflag.LetOpts
+	argsCpy := flagutil.GetArgsCopy(cmd)
+	args, err := flagutil.ParseArgsCleaned("LET", &opts, argsCpy)
+	if err != nil {
+		return errors.Wrap(err, "failed to parse LET args")
+	}
+	if len(args) != 3 {
+		return errors.New("failed to parse LET args")
```

Comment:
not sure if this is actually the syntax that we need here, but maybe there's a way to include the source location for this error too?
```suggestion
		return wrapError(errors.New("failed to parse LET args"), cmd.SourceLocation)
```

Reply:
actually.....

```
return newError(with.Command.SourceLocation,"failed to parse LET args")
```

Reply:
there's a number of places in this PR where we could do this.